### PR TITLE
[WIP] support for taxid filtering

### PR DESCRIPTION
**Pull request by hexylena**
Will probably merge into #123 but putting this here before I forget about it / open for discussion. I'd like to make use of taxid filtering, and this seems to be the supported way.

### CODE REVIEW

File: tools/ncbi_blast_plus/ncbi_macros.xml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
     <xml name="parallelism">
         <!-- If job splitting is enabled, break up the query file into parts -->
         <parallelism method="multi" split_inputs="query" split_mode="to_size" split_size="1000" merge_outputs="output1" />
     </xml>
 
     <xml name="preamble">
         <requirements>
-            <requirement type="package" version="2.7.1">blast</requirement>
+            <requirement type="package" version="2.9.0">blast</requirement>
+            <requirement type="package" version="13.3">entrez-direct</requirement>
```
Comment (peterjc) [inline tools/ncbi_blast_plus/ncbi_macros.xml:11]: Does BLAST need Entrez Direct for the taxonomy look up?

**Comment by peterjc** [comment]
Good idea - I've been wanting to work on this which is one of the new things added to local searches with the recent updates to BLAST.

**Comment by peterjc** [comment]
See also #123 for updating to BLAST 2.9.0

File: tools/ncbi_blast_plus/ncbi_macros.xml (hunk 1 above)
Comment (hexylena) [reply tools/ncbi_blast_plus/ncbi_macros.xml:11]: Yes, unfortunately. The script ships with blast itself, but relies on esearch and friends. 

> Note: this script depends on EDirect and it only works on Linux and Mac

via https://www.ncbi.nlm.nih.gov/books/NBK546209/

**Comment by bernt-matthias** [comment]
Sorry that I missed to check this .. but you may have a look at https://github.com/peterjc/galaxy_blast/pull/126

**Comment by hexylena** [comment]
Cool, glad you take it over @bernt-matthias

**Pull request closed without merging**